api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/ReportClaimMessageProcessor.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dhsc.htbhf.claimant.message.processor;
+
+import uk.gov.dhsc.htbhf.claimant.entity.Message;
+import uk.gov.dhsc.htbhf.claimant.message.MessageStatus;
+import uk.gov.dhsc.htbhf.claimant.message.MessageType;
+import uk.gov.dhsc.htbhf.claimant.message.MessageTypeProcessor;
+
+/**
+ * Responsible for reporting new and updated claims for MI reporting purposes.
+ * Fetches location data for a claim if it doesn't already exist, then reports the claim.
+ */
+public class ReportClaimMessageProcessor implements MessageTypeProcessor {
+
+    @Override
+    public MessageStatus processMessage(Message message) {
+        // TODO DW HTBHF-2412 Fetch location data for a claim and report the claim.
```

Comment:
Shouldn't this be a different ticket number, HTBHF-2412 describes creating a no-op implementation only?

Reply:
Corrected to HTBHF-2009